### PR TITLE
DRY out amsr2 bt params

### PR DESCRIPTION
**Pull request by trey-stafford**
DRY out the AMSR2 BT params. I expect that how we define/keep track of these params will continue to evolve but this makes it more clear that 'our' parameters differ from those provided by Goddard only  w/ the weather filter.

https://github.com/nsidc/pm_icecon/pull/18 adds tests to ensure that we don't further tweak either of these parameter sets without being aware of the potential consequences.

### CODE REVIEW

File: pm_icecon/bt/params/amsr2.py [hunk 1]
```diff
@@ -1,84 +1,67 @@
-"""amsr2.py: bootstrap parameters file.
```
Comment (trey-stafford) [inline pm_icecon/bt/params/amsr2.py:1]: I'd like to add more information about when these parameters are most appropriately used!

File: pm_icecon/bt/params/amsr2_goddard.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-"""amsr2_goddard.py: bootstrap parameters file.
```
Comment (trey-stafford) [inline pm_icecon/bt/params/amsr2_goddard.py:1]: I'd like to add more information about when these parameters are most appropriately used!

**Review by sc0tts — APPROVED** [review]
Yes!  I like having the chain-of-inheritance spelled out specifically.

**Comment by trey-stafford** [comment]
Merging this into [update_for_nise_cdr_cetb](https://github.com/nsidc/pm_icecon/tree/update_for_nise_cdr_cetb). I think it'll help us untangle some of the work on bt parameter sets we need to do tomorrow.

**Pull request merged**